lib/matplotlib/legend.py

```diff
@@ -279,6 +283,24 @@ def __init__(self, parent, handles, labels,
             setattr(self, name, value)
         del locals_view
 
+        # If scatter_uni_size is set, use a custom handler with set size.
+        if scatter_uni_size:
+            # See if custom handler has been provided for PathCollection
+            if (self._custom_handler_map is not None and
+                    PathCollection in self._custom_handler_map):
+                warnings.warn("'scatter_uni_size' was set, but custom " +
+                              "handler was provided. The former will " +
```

Comment:
Please be explicit, I would say "'scatter_uni_size will be ignored'".  (This is mostly because I personally always have to stop and think about which one is 'former' and which one is 'later').